pkg/proxy/util/conntrack.go

```diff
@@ -56,3 +57,34 @@ func ExecConntrackTool(execer exec.Interface, parameters ...string) error {
 	}
 	return nil
 }
+
+// ClearUDPConntrackForPort uses the conntrack tool to delete the conntrack entries
+// for the UDP connections specified by the port.
+// When a packet arrives, it will not go through NAT table again, because it is not "the first" packet.
+// The solution is clearing the conntrack. Known issues:
+// https://github.com/docker/docker/issues/8795
+// https://github.com/kubernetes/kubernetes/issues/31983
+func ClearUDPConntrackForPort(execer exec.Interface, port int) {
+	glog.V(2).Infof("Deleting conntrack entries for udp connections")
+	if port > 0 {
+		err := ExecConntrackTool(execer, "-D", "-p", "udp", "--dport", strconv.Itoa(port))
```

Comment:
I'm just observing that this will clear conntrack record it maybe should not.  We might have multiple services using the same port, and this will clear conntrack for all of them.

E.g. 
- client A -> service A port 12345
- client B -> service B port 12345
- this runs for 12345
- this clears both

It's not clear this is right.  What do you think?